### DISCUSSION TITLE
how to generate breadcrumbs for one-level menu

### DISCUSSION THREAD

**Discussion by whg517** [refinedev/refine#5108] [Q&A] [answered]
I noticed that in the antd breadcrumbs component, there is a logic that when the component is routed to level 1, no breadcrumbs will be generated. Is this really reasonable? If I have multiple tier 1 routes and the top one is dashboard, then I will never be able to use breadcrumbs.

https://github.com/refinedev/refine/blob/dd7483e9ef4d7033e2d53ca47935f9d2b1023ba7/packages/antd/src/components/breadcrumb/index.tsx#L41-L43

![image](https://github.com/refinedev/refine/assets/21030747/fc637529-f9fa-4ace-8a33-c61916b97bea)

![image](https://github.com/refinedev/refine/assets/21030747/ffea5002-cbe0-460d-a13a-329050a05ade)

**Comment by aliemir** [comment, marked as answer]
Hey @whg517, like you've spotted in the source yourself, we have a condition to remove breadcrumb if it only has one level. We're thinking this is a more common approach 🤔 But definitely we can have a poll or something to find out what the community is thinking about it 😅 

Luckily, we have an option to customize the breadcrumb easily by swizzling and assigning it globally to replace it in every component 🚀 

Start with swizzling the `Breadcrumb` component, check out the swizzle command guide here https://refine.dev/docs/packages/documentation/cli/#usage

After you customize your component, you can use the `options.breadcrumb` prop of `<Refine>` component to override that component globally. Check out the docs here https://refine.dev/docs/api-reference/core/components/refine-config/#breadcrumb

Let us know if you have any issues with the swizzle or with the global config 🙏

**Reply by whg517** [reply]
Now that I remember, there is a global breadcrumb that can be passed. I will take custom breadcrumbs and then use them globally. Thank you very much.
